webhooks/virtualmachine/validation/controller_validator.go

```diff
@@ -0,0 +1,220 @@
+// © Broadcom. All Rights Reserved.
```

Comment:
Rename this file to `virtualmachine_validator_hardware_controllers.go`. Do the same for the associated test file. Following patterns in a project is important.